pages/linux/debtap.md

```diff
@@ -0,0 +1,24 @@
+# debtap
+
+> Convert .deb packages into Arch Linux packages.
+> More information: <https://github.com/helixarch/debtap>
+
+- Update debtap database (before first run)
+
+`sudo debtap -u`
+
+- Convert a package
+
+`debtap {{package.deb}}`
+
+- Convert a package (bypass all questions, not recommended)
+
+`debtap -Q {{package.deb}}`
+
+- Then install with pacman
+
+`sudo pacman -U {{package.pkg.tar.zst}}`
+
+- Generate PKGBUILD file
+
+`debtap -P {{package.deb}}`
```

Comment:
We use `path/to/file` as placeholder
```suggestion
`debtap --pkgbuild {{path/to/package.deb}}`
```

Reply:
We added the same suggestions. 😄

Reply:
Reviewers attacking a first-time contributor. :heart:

Reply:
Than you both for your advice 😂